index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio-Project</title>
+</head>
+
+<body>
+    <nav class="navbar">
+        <a class="logo" href="#">My Logo</a>
+        <div class="navbarItems">
+            <a href="#">Portoflio</a>
+            <a href="#">About</a>
+            <a href="#">Contact</a>
+        </div>
+        <img class="hamburger" src="./Assets/images/bars.svg" alt="hamburger">
+    </nav>
```

Comment:
- [ ] Kindly consider taking a `header` tag directly under `body` tag. that `header` tag would have two children : an `a` tag for logo and a `nav` tag that would consist of the hamburger menu. the reason for that is to follow best practice and conventions, better semantics, and better user experience.

Reply:
Hi @Zuhaib042,

I have made the changes asked as per your comment and requesting for the review again.

Thank you for valuable feedback

Regards